docs/sql/statements/create-repository.rst

```diff
@@ -669,6 +669,50 @@ Parameters
 
   The container path to use for snapshots.
 
+.. _sql-create-repo-gcs-compress:
+
+**compress**
+  | *Type:*    ``boolean``
+  | *Default:* ``true``
+
+  Whether CrateDB should compress the metadata part of the snapshot or not.
+
+  CrateDB does not compress the actual table data.
+
+.. _sql-create-repo-gcs-chunk_size:
+
+**chunk_size**
+  | *Type:*    ``bigint`` or ``text``
+  | *Default:* ``null``
+
+  Defines the maximum size of any single file that comprises the snapshot. If
+  set to ``null``, CrateDB will not split big files into smaller chunks.  You
+  can specify the chunk size with units (e.g., ``1g``, ``5m``, or ``9k``). If
+  no unit is specified, the unit defaults to bytes.
+
+.. _sql-create-repo-gcs-connect_timeout:
+
+**connect_timeout**
+  | *Type:*    ``text``
+  | *Default:* ``20 seconds``
+
+  Defines the timeout to establish a connection to the Google Cloud Storage
+  service. The value should specify the unit. For example, a value of 5s
+  specifies a 5 second timeout. The value of -1 corresponds to an infinite
+  timeout. The default value is 20 seconds.
```

Comment:
Maybe we can skip the `default value` as it's clear right above, but also doesn't harm to keep it, but it's not consistent with the other settings above where this sentence is missing.

Reply:
So what is the final suggestion?